packages/plugin-ext/src/hosted/node/hosted-plugin-process.ts

```diff
@@ -120,13 +121,22 @@ export class HostedPluginProcess implements ServerPluginRunner {
             }
         });
         const hostedPluginManager = rpc.getProxy(MAIN_RPC_CONTEXT.HOSTED_PLUGIN_MANAGER_EXT);
+        this.killChildProcesses(cp.pid);
```

Comment:
it should go in `then` case after after plugin got unloaded

Reply:
Done